README.md

```diff
@@ -76,7 +76,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2.0.0
       - name: Self-hosted Renovate
-        uses: renovatebot/github-action@v21.30.0
+        uses: renovatebot/github-action@v26.4.1
```

Comment:
Should these types of changes now be done automatically after your other PR was merged?

Reply:
I agree. I removed fbf10e6d3cb538e9a26ab901d8f9e27834409a31 .

Reply:
👍 The pull request was created as expected. #559